src/lang/po/fr/Prusa-Firmware-Buddy_fr.po

```diff
@@ -1200,11 +1200,11 @@ msgstr ""
 "d'installation pour "
 "Original Prusa MINI. "
 "Voulez-vous "
-"continuer ?"
+"continuer?"
 
 #: src/gui/test/screen_test_msgbox.c:28
 msgid "Welcome to the Original Prusa MINI setup wizard. Would you like to continue?"
-msgstr "Bienvenue sur l'assistant de configuration de l'Original Prusa MINI. Voulez-vous continuer ?"
+msgstr "Bienvenue sur l'assistant de configuration de l'Original Prusa MINI. Voulez-vous continuer?"
```

Comment:
Do you really want to strip the space in front of the '?' here? What are the French typographical rules for using or not using a space in front of '?' or '!' ?

Reply:
I fully agree with the spaces with ""?" but to make the whole translation homogeneous it is better to remove the space.

Reply:
Isn't it pretty that "!" who walks alone?
![WP_20200727_23_10_05_Pro](https://user-images.githubusercontent.com/64924478/88593047-3f424680-d05f-11ea-87f3-c845577f6ebf.jpg)

Reply:
> Do you really want to strip the space in front of the '?' here? What are the French typographical rules for using or not using a space in front of '?' or '!' ?

and honestly we French it does not bother us if it lacks a space.

Reply:
another example, it's not too much in its place on "!"
![WP_20200727_23_00_35_Pro](https://user-images.githubusercontent.com/64924478/88595073-a31a3e80-d062-11ea-85f2-a125b8124630.jpg)

Reply:
@carlin57 Yes, the space before '?' and '!' makes things a bit difficult, but not something we can't handle in the 32 bit. We'd like to keep the spaces wherever possible, since it is the preferred way the French language is written (as I've been told)

Reply:
then in this case if you prefer to keep the spaces, we can try to rework the translations so that the "!" not walk alone.